.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "content/measurement-kit"]
+	path = content/measurement-kit
+	url = https://github.com/measurement-kit/measurement-kit.git
```

Comment:
Is this a good idea? Sub-repositories are always a bit complex to manage, especially when you want to change branch inside them. But maybe that's not going to happen in our case?